staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -762,27 +746,6 @@ func TestCacherNoLeakWithMultipleWatchers(t *testing.T) {
 	}
 }
 
-func TestWatchInitializationSignal(t *testing.T) {
```

Comment:
Why did you remove this test?

Reply:
@wojtek-t - mainly because this is already a part of `cacher_test.go`: https://github.com/kubernetes/kubernetes/blob/4c40d749006a8895f6718f2b624a3dbe975988ab/staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go#L383

From both the implementations, at least based on what it is testing for, I thought it was semantically equivalent and hence could be removed. The only difference I could see is the new test (using the common tester) is propagating an object and creating a watch against the created object's rv, and the old test is creating one against `0`. Am I missing something?

Reply:
Ok - you're right. I forgot about that.